clients/src/main/java/org/apache/kafka/common/security/oauthbearer/internals/secured/ConfigurationUtils.java

```diff
@@ -228,4 +233,16 @@ public <T> T get(String name) {
         return (T) configs.get(name);
     }
 
+    // make sure the url is in the "org.apache.kafka.sasl.oauthbearer.allowed.urls" system property
+    public void throwIfURLIsNotAllowed(String urlConfig) {
+        Set<String> allowedLoginModuleList = Arrays.stream(
+                        System.getProperty(ALLOWED_SASL_OAUTHBEARER_URLS_CONFIG, "").split(","))
```

Comment:
Maybe we should create `ALLOWED_SASL_OAUTHBEARER_URLS_DEFAULT` instead of having `""` here. WDYT?

Reply:
Sounds good!